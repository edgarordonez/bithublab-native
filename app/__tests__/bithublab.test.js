// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import BitHubLab from '../bithublab';

it('renders without crashing', () => {
  const rendered = renderer.create(<BitHubLab />);
  expect(rendered).toMatchSnapshot();
});
