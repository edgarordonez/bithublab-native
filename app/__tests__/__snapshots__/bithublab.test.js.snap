// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<RCTSafeAreaView
  style={
    Object {
      "backgroundColor": "#FAFAFA",
      "flex": 1,
      "paddingTop": 0,
    }
  }
/>
`;
